jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -686,7 +690,7 @@ private String(String s, char c) {
 
 			helpers.putByteInArrayByIndex(value, slen, (byte) c);
 		} else {
-			value = new byte[concatlen * 2];
+			value = StringUTF16.newBytesFor(concatlen);
```

Comment:
Just noting that this looks like a small method and should get inlined, but it is not a guarantee. @cathyzhyi should we consider adding this to the always worth inlining list?

Reply:
opened an issue for adding the method to the list https://github.com/eclipse/openj9/issues/8831